src/ProjectConfiguration.ts

```diff
@@ -617,6 +624,50 @@ export function validateCompanionDefaultWakeInterval(
   return diagnostics;
 }
 
+export function validateTileBuildTarget( tile: Tile, { buildTargets }: AppProjectConfiguration) {
+  if (tile.buildTargets !== undefined){
+    const diagnostic = constrainedSetDiagnostics({
+      actualValues: tile.buildTargets,
+      knownValues: buildTargets,
+      valueTypeNoun: 'tile build targets',
+      notFoundIsFatal: true
+    })
+    return diagnostic;
+  }
+
+  // Return empty list. Nothing to check
+  return new DiagnosticList();
+}
+
+export function validateTileUUID(tile: Tile, config: AppProjectConfiguration) {
+  const diagnostic = new DiagnosticList()
```

Comment:
Is your auto-formatter running? It should be running via a Git hook to add a semi-colon on the end of the line like here.

Reply:
I don't think the auto-formatter is running. I'll make sure to run it before updating the PR.

Reply:
Ran `yarn lint && yarn checklist` and they report no problems.

Reply:
I have no git hook active. Should we maybe enable them?

Reply:
I think it's an oversight on my part, I upgraded Husky the tool that manages the Git hooks at some point with other dependencies, but they changed the behaviour to no longer auto-install the hook. I already had the hook so I didn't notice. Fixed here: https://github.com/Fitbit/fitbit-sdk-toolchain/pull/277